shared/middleware/helpers/storage_apps.rb

```diff
@@ -45,6 +48,20 @@ def get(channel_id)
     row = @table.where(id: id).exclude(state: 'deleted').first
     raise NotFound, "channel `#{channel_id}` not found" unless row
 
+    # For some apps, if it was created by a signed out user, we don't want anyone
+    # else to be able to access it (for privacy reasons)
+    anonymous_age_restricted_apps = ['applab', 'gamelab', 'weblab']
```

Comment:
don't love having apps hardcoded here, but not sure there's really a better approach